esmvalcore/_task.py

```diff
@@ -303,8 +303,8 @@ def _initialize_cmd(self):
             if self.settings['profile_diagnostic']:
                 profile_file = Path(self.settings['run_dir']) / 'profile.bin'
```

Comment:
```suggestion
                profile_file = Path(self.settings['run_dir']) / 'profile.json'
```
because vprof outputs json